modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,85 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
```

Comment:
The `Openshift Update Service graph data container image has already been created and pushed to a repository accessible to the OpenShift Update Service` is also a prerequisite.

Reply:
Fixed